server/src/main/java/com/feedoong/server/global/exception/ExceptionType.java

```diff
@@ -21,6 +21,8 @@ public enum ExceptionType {
 	INVALID_PASSWORD(BAD_REQUEST, "올바르지 않은 비밀번호 입니다."),
 	INVALID_TOKEN_TYPE(BAD_REQUEST, "올바르지 않은 토큰 타입 입니다."),
 	INVALID_REFRESH_TOKEN(BAD_REQUEST, "사용할 수 없는 토큰 입니다."),
+	JSOUP_CONNECTION_FAILED(BAD_REQUEST, "Jsoup URL 연결에 실패했습니다."),
+	RSS_URL_NOT_FOUND(BAD_REQUEST, "RSS 주소를 찾을 수 없습니다."),
```

Comment:
Jsoup은 서버 내부에서 사용하는 거라서 프론트에 뿌려줄 메시지에는 포함시킬 필요 없어 보이고, 제대로 된 URL이 아니라서 발생하는 문제라면 아래처럼 수정하면 좋을 듯.
```suggestion
	INVALID_URL(BAD_REQUEST, "잘못된 URL입니다."),
	RSS_FEED_NOT_FOUND(BAD_REQUEST, "RSS 주소를 찾을 수 없습니다."),
```